index.html

```diff
@@ -0,0 +1,128 @@
+<html>
+  <head>
+    <title>Dr. Norman Borlaug</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main id="main">
+      <header>
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+      </header>
+      <figure id="img-div">
+        <img
+          id="image"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          alt="People working"
+        />
+        <p id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </p>
+      </figure>
+      <div id="tribute-container">
+        <div id="header_space">
+          <h3 id="tribute-header">Here's a time line of Dr. Borlaug's life:</h3>
+        </div>
+        <ul id="tribute-info">
+          <li class="tribute-element">
+            <strong>1914</strong> - Born in Cresco, Iowa
+          </li>
+          <li class="tribute-element">
+            <strong>1933</strong> - Leaves his family's farm to attend the
+            University of Minnesota, thanks to a Depression era program known as
+            the "National Youth Administration"
+          </li>
+          <li class="tribute-element">
+            <strong>1935</strong> - Has to stop school and save up more money.
+            Works in the Civilian Conservation Corps, helping starving
+            Americans. "I saw how food changed them", he said. "All of this left
+            scars on me."
+          </li>
```

Comment:
## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.
- [OPTIONAL] I think It should be better if you put the text into the tag `<p>` because it's appropriate to improve the accessibility, SEO, and overall readability of your code.